.github/workflows/deploy.yml

```
@@ -17,58 +14,72 @@ jobs:
   build-and-push:
     if: github.event_name == 'push' || github.event.pull_request.merged == true
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)